source/configuration_manual/sql.rst

```diff
@@ -0,0 +1,527 @@
+The connect_string is a space separated list of ``key=value`` pairs.
+
+Cassandra
+=========
+
+Driver name ``cassandra``.
+
+Driver for Apache Cassandra CQL server.
+
+To compile support for this driver, you need to have DataStax C/C++ driver and headers installed.
+
+Supported options
+-----------------
+
+``host``
+--------
+- Default: <empty>
+- Values: :ref:`string`
+
+Host or IP address to connect. Can appear multiple times.
+
+``port``
+--------
+- Default: ``9042``
+- Values: :ref:`uint`
+
+RPC port to use.
+
+``keyspace``
+------------
+- Default: <empty>
+- Values: :ref:`string`
+
+Specifies the keyspace name to use.
+
+``dbname``
+----------
+
+Alias for ``keyspace``.
+
+``version``
+-----------
+- Default: Depends on driver version.
+- Values: ``v3``, ``v4``, ``v5`` (refer to cassandra manual).
+
+Cassandra driver version to use. It is good idea to specify this to avoid warnings about version handshake.
+If you want to use server-side prepared statements, you need to use at least ``v4``.
+
+``user``
+--------
+- Default: <empty>
+- Values: :ref:`string`
+
+Username for authentication.
+
+``password``
+------------
+- Default: <empty>
+- Values: :ref:`string`
+
+Password for authentication.
+
+``ssl_ca``
+----------
+- Default: <empty>
+- Values: :ref:`string`
+
+Path to SSL certificate authority file to use to validate peer certificate.
+
+``ssl_cert_file``
+-----------------
+- Default: <empty>
+- Values: :ref:`string`
+
+Path to a certificate file to use for authenticating against the remote server.
+
+``ssl_private_key_file``
+------------------------
+- Default: <empty>
+- Values: :ref:`string`
+
+Path to private key matching ``ssl_cert_file`` to use for authenticating against the remote server.
+
+``ssl_verify``
+---------------
+- Default: ``none``
+- Values: ``none``, ``cert``, ``cert-ip``, ``cert-dns``
+
+Configure the peer certificate validation method.
+
+``none``
+  Disables validation
+
+``cert``
+  Validate that the certificate is valid.
+
+``cert-ip``
+  Validate that the certificate is valid and has Common Name or Subject Alternate Name for the IP address.
+
+``cert-dns``
+   Validate that the certificate is valid and has Common Name or Subject Alternate Name that matches PTR resource record for the server's IP address.
+
+``log_level``
+--------------
+- Default: ``warn``
+- Values: ``critical``, ``error``, ``warn``, ``info``, ``debug``, ``trace``
+
+Driver log level.
+
+``debug_queries``
+-----------------
+- Default: no
+- Values: :ref:`boolean`
+
+Whether to log queries.
+
+``metrics``
+-----------
+- Default: no
+- Values: :ref:`boolean`
+
+Whether to collect metrics.
+
+.. seealso:: :ref:`cassandra_metrics_json_output`
+
+``num_threads``
+----------------
+- Default: <driver dependant>
+- Values: :ref:`uint`
+
+Set number of IO threads to handle query requests.
+
+``page_size``
+-------------
+- Default: -1
+- Values: :ref:`uint`
+
+Set the number of results per page.
```

Comment:
`-1 = Disabled.` Would be useful also to describe the purpose of it .. but I'm not sure where to easily find a good clear explanation.